src/Dynatrace.OpenTelemetry.Exporter.Metrics/Dynatrace.OpenTelemetry.Exporter.Metrics.csproj

```diff
@@ -6,7 +6,7 @@
     <Company>Dynatrace</Company>
     <Product>Dynatrace OpenTelemetry Metrics Exporter for .NET</Product>
     <PackageId>Dynatrace.OpenTelemetry.Exporter.Metrics</PackageId>
-    <Version>0.4.0-beta</Version>
+    <Version>0.5.0-beta</Version>
```

Comment:
I guess we want to make this a stable release, right?

Reply:
I am not sure how this is usually done in .NET, but should we mirror the version numbers of the .NET sdk itself?

Reply:
I think we can leave it like this in terms of versioning for now (but I think I agree with the comment about removing `-beta`).

Reply:
changed it back to 0.5.0, we'll create a PR to release 1.0 (non-beta).